content/users/datahub/links/_index.md

```diff
@@ -7,15 +7,15 @@ weight: 40
 ---
 
 - EGI DataHub
-  - <https://datahub.egi.eu/>
-  - Presentation: <http://go.egi.eu/datahub-intro>
+  - https://datahub.egi.eu/
+  - Presentation: http://go.egi.eu/datahub-intro
   - Community Forum:
-    <https://community.egi.eu/c/egi-services/datahub>
-  - <https://wiki.egi.eu/wiki/EGI_Federated_Data>
+    - https://community.egi.eu/c/egi-services/datahub
+  - https://wiki.egi.eu/wiki/EGI_Federated_Data
 - System requirements
-  - <https://onedata.org/docs/doc/system_requirements.html>
+  - https://onedata.org/docs/doc/system_requirements.html
 - Official Onedata documentation
-  - <https://onedata.org>
-  - <https://onedata.org/#/home/documentation>
-  - Getting started: <https://github.com/onedata/getting-started>
-  - Source code: <https://github.com/onedata>
+  - https://onedata.org
+  - https://onedata.org/#/home/documentation
+  - Getting started: https://github.com/onedata/getting-started
+  - Source code: https://github.com/onedata
```

Comment:
I don't like much URLs by themselves, I would do something like:
- [EGI DataHub](https://datahub.egi.eu)
   - [Presentation](http://go.egi.eu/datahub-intro)
...

Reply:
I'm not always a fan of hidden tlinks, but I updated them :)